libs/journeys/ui/src/components/StepFooter/FooterButtonList/ReactionButton/ReactionButton.spec.tsx

```diff
@@ -0,0 +1,19 @@
+import { render } from '@testing-library/react'
+import { ReactionButton } from '.'
+
+jest.mock('@mui/material/useMediaQuery', () => ({
+  __esModule: true,
+  default: () => true
+}))
+
+describe('ReactionButton', () => {
+  it('should render thumbs up or like button', () => {
```

Comment:
`or like button` makes it sound like theres 2 possible renderable elements
```suggestion
  default: () => true
}))

describe('ReactionButton', () => {
  it('should render thumbs up button', () => {
```